src/components/pluralize.cjsx

```diff
@@ -0,0 +1,18 @@
+React = require 'react'
+
+# TODO find a better pluralizer
```

Comment:
That looks great! @philschatz i remember you were hesitant about adding something like this --[underscore string](https://github.com/epeli/underscore.string) -- because we didn't need it at the time.  If I switch to either of these, we can replace the camelcase library as well.  Should I switch out for underscore inflections or underscore string now?